_includes/br/The-shell-aka-REPL.md

```diff
@@ -1,13 +1,13 @@
-|                                        |                              |
-| -------------------------------------- | ---------------------------- |
-| Recall last result                     | `ans`                        |
-| Interrupt execution                    | \[Ctrl\] + \[C\]             |
-| Clear screen                           | \[Ctrl\] + \[L\]             |
-| Run program                            | `include("filename.jl")`     |
-| Get help for `func` is defined         | `?func`                      |
-| See all places where `func` is defined | `apropos("func")`            |
-| Command line mode                      | `;`                          |
-| Package Manager mode                   | \]                           |
-| Help mode                              | ?                            |
-| Exit special mode / Return to REPL     | \[Backspace\] on empty line  |
-| Exit REPL                              | `exit()` or \[Ctrl\] + \[D\] |
+|                                               |                              |
+| --------------------------------------------- | ---------------------------- |
+| Relembrar o último resultado                  | `ans`                        |
```

Comment:
"Recall" here is not used in the metaphoric sense of remembering, but rather as retrieving, fetching something back, invoking again. Maybe a better translation would be "reutilizar"?

Reply:
You are right, "reutilizar" would be a better translation

Reply:
You're right, "reutilizar" would be a better translation.